doc/adv-manual/risk-features.rst

```diff
@@ -49,12 +58,16 @@ Wood       Wood Type B  0.3
 Wood       Wood Type C  0.5
 ========= ============ =======
 
-The engine will read such file and when performing the risk calculation
-will use all three kinds of risk functions and compute a single result
+The engine will read this mapping file and when performing the risk calculation
+will use all three kinds of risk functions to compute a single result
 with a weighted mean algorithm. The sums of the weights must be 1
 for each exposure taxonomy, otherwise the engine will raise an error.
 In this case the taxonomy mapping file works like a risk logic tree.
 
 Internally both the first usage and the second usage are treated in
 the same way, since the first usage is a special case of the second
-when all the weights are equal to 1
+when all the weights are equal to 1.
+
+Currently, this taxonomy mapping feature is available for the scenario
+and event based risk calculators. The feature hasn't been added to the
+classical damage and risk calculators.
```

Comment:
Actually it has not been tested for classical_damage and classical_risk but it should work even for those calculators, at least I think.